testing/toxcore/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Maintainer: Jonathan Sieber <mail@strfry.org>
+
+pkgname=toxcore
+pkgver=0.2.2
+pkgrel=0
+pkgdesc="Tox communication project - Core Library"
+url="https://tox.chat/"
+arch="all"
+license="GPL"
+makedepends="linux-headers cmake libsodium-dev opus-dev libvpx-dev"
+subpackages="$pkgname-dev $pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/TokTok/c-toxcore/archive/v$pkgver.tar.gz"
+
+# Toxcore's tests do require networking and are not working properly in Travis-CI
+options="!check"
+
+builddir="$srcdir/c-$pkgname-$pkgver"
+
+prepare() {
+	cd "$builddir"
+	cmake . \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_INSTALL_LIBDIR=lib \
+		|| return 1
```

Comment:
cmake needs to run in build.
There is also no need to return 1 please remove it.